src/main/java/de/tum/in/www1/artemis/service/FileService.java

```diff
@@ -433,8 +433,8 @@ public String actualPathForPublicPath(String publicPath) {
      *
      * @param actualPathString the path to the file in the local filesystem
      * @param entityId         the id of the entity associated with the file
-     * @throws FilePathParsingException if the path is unknown
      * @return the public file url that can be used by users to access the file from outside
+     * @throws FilePathParsingException if the path is unknown
```

Comment:
Why doesn't the signature contain this exception?

Reply:
removed because unrelated to my changes.